extensions/parquet/table/src/main/java/io/deephaven/parquet/table/TypeInfos.java

```diff
@@ -376,11 +378,33 @@ public PrimitiveBuilder<PrimitiveType> getBuilder(boolean required, boolean repe
             if (!isValidFor(dataType)) {
                 throw new IllegalArgumentException("Invalid data type " + dataType);
             }
+            // Write instants as Parquet TIMESTAMP(isAdjustedToUTC = true, unit = NANOS)
             return type(PrimitiveTypeName.INT64, required, repeating)
                     .as(LogicalTypeAnnotation.timestampType(true, LogicalTypeAnnotation.TimeUnit.NANOS));
         }
     }
 
+    private enum LocalDateTimeType implements TypeInfo {
+        INSTANCE;
+
+        private static final Set<Class<?>> clazzes = Collections.singleton(LocalDateTime.class);
+
+        @Override
+        public Set<Class<?>> getTypes() {
+            return clazzes;
+        }
+
+        @Override
+        public PrimitiveBuilder<PrimitiveType> getBuilder(boolean required, boolean repeating, Class<?> dataType) {
+            if (!isValidFor(dataType)) {
+                throw new IllegalArgumentException("Invalid data type " + dataType);
+            }
```

Comment:
We copy this everywhere. Is there a way to move it up to an interface (defaulted), or at least cut it down to a one-liner?